### PR TITLE
Dbms mongodb1

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**

### CODE REVIEW

**Review by udeysingh95 — COMMENTED** [review]
Loved the work! 
Some improvements:
- [x] English mistakes in 1-get-started.md
- [x] GEt - needs to be Get 
- [x] lets maintain the consistency of README file in root directory. I see that you added some sub-chapters in that - lets try to explain the content of that whole chapter 9 together in a single paragraph like we have been doing for other chapters. That doesn't mean you can summarize the whole chapter right now and never touch it again. it will be iterative only and the content of that summary will keep getting updated with sub-chapters that you will create in `9-mongodb`directory.

PR is looking very good.

File: 5-dbms/9-mongodb/1-get-started.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+  Now u will see a box at the right side of your screen named Available Downloads. First the Version dropdown, select the latest version of MongoDB to download. In the Platform dropdown, select the operating system u are going to install. In the Package dropdown, select msi. Check all the options u selected twice before you click download.
+  
+  <p align="center"><img src="https://user-images.githubusercontent.com/54719422/91629726-0cd29300-e9e9-11ea-9880-6f37bc7754c7.png" height="" width=""></p> 
+
+So now you are all set to download MongoDB, just click on download button and the downloading will get started.
+
+## Step-3 Installing MongoDB
+
+This is the most important part of the whole process. U have to follow the below mentioned steps very carefully. So now as you have successfully downloaded the MongoDB its time to install it.
```
Comment (udeysingh95) [inline 5-dbms/9-mongodb/1-get-started.md:24]:
```suggestion
This is the most important part of the whole process. You have to follow the below-mentioned steps very carefully. So now as you have successfully downloaded the MongoDB its time to install it.
```

Comment (udeysingh95) [reply 5-dbms/9-mongodb/1-get-started.md:24]: What is this "U"! Stick to semi-formal writing style and not texting.

Comment (udeysingh95) [reply 5-dbms/9-mongodb/1-get-started.md:24]: Check for the similar issues through-out document.

File: 5-dbms/9-mongodb/1-get-started.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+e) Now save and exit the editor press "esc" and then type **:wq!** as shown
+<img src="https://miro.medium.com/max/700/1*3JX8Tg3MxhoPqC65r1pfew.png" height="" width="">
+
+g) Congratulations! you have successfully completed the installation process.
+
+## Step-5  Verify That Setup was Successful
+a) Close down the current Hyper terminal and quit the application.\
+b) Re-launch Hyper
+c) Type the following commands into the Hyper terminal:
```
Comment (udeysingh95) [inline 5-dbms/9-mongodb/1-get-started.md:94]: This point `c` is falling in the same line as `b` and the content is also looking a bit congested in here.

**Review by udeysingh95 — CHANGES_REQUESTED** [review]
### Summary is good.
Code like formatting could be used around all the commands included in this document. I see that there are a lot of commands lying in between the text - that can enjoy some highlighting. Also, some images (with black backgrounds) says a very little and take a lot of space - if it is fine - how about adding the content in code-snipped there as well - rather than blocking such a big block of space with almost no information.

**Comment by Vaibhavpratapsingh22** [comment]
@udeysingh95  what i think is that through images it gets easier for the user. As the user can relate his and my screen and if he sees the same user gets relaxed. But in the case of code snippets it will get little tricky for a new user to understand.
If u like, i can minimise the size of the images.

What u say??

**Comment by udeysingh95** [comment]
If you think, its more relatable - then let it be like this only. Maybe adjust some spacing around it a bit. They are lying in between text market at the moment. Maybe it will look a bit more elegant then.

**Comment by Vaibhavpratapsingh22** [comment]
Cool then @udeysingh95 ,  I will edit the images

**Review by udeysingh95 — APPROVED** [review]

**Comment by giteshk11** [comment]
This looks good, good job @Vaibhavpratapsingh22 

**FYI:** Installation process always changes over time so it becomes hard to maintain the installation process. Plus official docs are always there to refer.

**Review by giteshk11 — APPROVED** [review]

File: 5-dbms/9-mongodb/1-get-started.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+Now in this new window type "mongo". This will run the MongoDB on your system.
+
+<img src="https://user-images.githubusercontent.com/54719422/91634489-7b761780-ea0e-11ea-86dc-d3d3eb26c344.png" height="" width="">
+
+So finally, now you can start playing with MongoDB commands.
+
+
+## Summary
+In this chapter we have completed the installation, setup the MongoDB on our local system.
```
Comment (SaarthakDeshmukh) [inline 5-dbms/9-mongodb/1-get-started.md:132]:
```suggestion
In this chapter we have completed the installation and setup of the MongoDB on our local system.
```

File: 5-dbms/README.md [hunk 4]
```diff
@@ -34,3 +34,4 @@ ORDBMS is a DBMS based on the Relational model and object-oriented database mode
 We will learn about AMD which includes Distributed Data Management System(DDBMS) and NoSQL. 
 
 ## Chapter 9 - MongoDB
+We will learn how to install MongoDB on your local system. Believe me, this is going to be very easy if you follow the all steps as discussed.
```
Comment (SaarthakDeshmukh) [inline 5-dbms/README.md:37]:
```suggestion
We will learn how to install MongoDB on your local system. Believe me, this is going to be very easy if you follow all the steps as discussed.
```

File: 5-dbms/9-mongodb/1-get-started.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+### 1. Choose Setup Type
+You can choose either the Complete (recommended for most users) or Custom setup type. The Complete setup option installs MongoDB and the MongoDB tools to the default location. The Custom setup option allows you to specify which executables are installed and where.
+
+### 2. Service Configuration
+Starting in MongoDB 4.0, you can set up MongoDB as a Windows service during the install or just install the binaries.
+
+<p align="center"><img src="https://user-images.githubusercontent.com/54719422/91629976-486e5c80-e9eb-11ea-88f4-77403fa9e75b.png" height-="" width=""></p>
+
+a) Select Install MongoDB as a Service MongoDB as a service.
```
Comment (SaarthakDeshmukh) [inline 5-dbms/9-mongodb/1-get-started.md:44]: Check out the spacing. Structure is bit congested here. a) and b) are coming in the same line plus, c) is getting an extra special line.

**Review by SaarthakDeshmukh — CHANGES_REQUESTED** [review]
Great Work! PR is looking in shape. There are few structure and grammatical changes which has to be taken care off.

**Review by SaarthakDeshmukh — APPROVED** [review]

**Pull request merged**